core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -125,41 +124,12 @@ public class GenericContainer<SELF extends GenericContainer<SELF>>
     /*
      * Default settings
      */
-    @NonNull
-    private LinkedHashSet<Integer> exposedPorts = new LinkedHashSet<>();
-
-    @NonNull
-    private List<String> portBindings = new ArrayList<>();
-
     @NonNull
     private List<String> extraHosts = new ArrayList<>();
```

Comment:
Revisit whether `extraHosts` should really be a field in `GenericContainer`, or if another place makes more sense (out of scope of this PR).